### ISSUE TITLE
Error with ignored file

### ISSUE THREAD

**Issue by pmcelhaney** [ataylorme/eslint-annotate-action#40]
The first item in my report looks like this.

```json
  {
    "filePath": "/Users/rpmcelbd/code/front-end/.github/workflows/web-ci.yaml",
    "messages": [
      {
        "fatal": false,
        "severity": 1,
        "message": "File ignored by default.  Use a negated ignore pattern (like \"--ignore-pattern '!<relative/path/to/filename>'\") to override."
      }
    ],
    "suppressedMessages": [],
    "errorCount": 0,
    "fatalErrorCount": 0,
    "warningCount": 1,
    "fixableErrorCount": 0,
    "fixableWarningCount": 0,
    "usedDeprecatedRules": []
  }
```

## Actual

I'm getting the following error.

```
Empty value for parameter 'output.annotations[0].end_line': undefined
```

## Expected

It should skip this warning because the file is ignored. It doesn't matter that there's no `end_line`. It should probably ignore anything that doesn't have a `ruleId`.

**Comment by ataylorme** [comment]
This should have been fixed in `v2`. Please re-open the issue if that is not the case.
